database/migrations/2022_11_28_200151_create_seasons_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('seasons', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('media_id');
+            $table->integer('value');
+            $table->string('title', 64);
+            $table->timestamps();
+
+            $table->foreign('media_id')->references('id')->on('medium')->onDelete('cascade');
```

Comment:
should be medium_id
$table->foreign('medium_id')->references('id')->on('media')->onDelete('cascade');
on('media') odnosi się do jakiej tabeli chcesz zrobić relację